src/test/kotlin/no/nav/tilleggsstonader/sak/utbetaling/iverksetting/IverksettDeprecatedClientTest.kt

```diff
@@ -0,0 +1,82 @@
+package no.nav.tilleggsstonader.sak.utbetaling.iverksetting
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.client.WireMock
+import com.github.tomakehurst.wiremock.client.WireMock.aResponse
+import com.github.tomakehurst.wiremock.client.WireMock.anyUrl
+import com.github.tomakehurst.wiremock.client.WireMock.get
+import com.github.tomakehurst.wiremock.client.WireMock.okJson
+import com.github.tomakehurst.wiremock.client.WireMock.post
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration
+import no.nav.security.mock.oauth2.http.objectMapper
+import no.nav.tilleggsstonader.libs.test.assertions.catchThrowableOfType
+import no.nav.tilleggsstonader.sak.utbetaling.iverksetting.IverksettDtoUtil.iverksettDto
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatCode
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.springframework.boot.web.client.RestTemplateBuilder
+import org.springframework.http.HttpStatus
+import org.springframework.web.client.HttpClientErrorException
+import org.springframework.web.client.RestTemplate
+import java.net.URI
+import java.util.UUID
+
+class IverksettDeprecatedClientTest {
```

Comment:
```suggestion
class IverksettClientTest {
```

Reply:
Dette er en tidligere IverksettClient som er i en annet package som jeg fant etter at jeg hadde lagt til klienten 😬 
Så for å ikke fjerne den og oppdatere kode (simulering finnes ikke ennå) så endret jeg bare navn på den till `IverksettDeprecatedClient`

Reply:
Ja, men er ikke testen for den nye klienten?

Reply:
Doh! Takk for att du passer på 💯